### ISSUE TITLE
Loading State_dict Error happend when do predict

### ISSUE THREAD

**Issue by jwee-star** [MAZiqing/FEDformer#37]
hi, thanks for your interesting work!
I train FEDformer in my dataset, and get really lower train and vali loss than Informer and Autoformer.
but when i want to do predict， i keep the same setting as training, a load state_dict error happend. i haven't solved the problem.
![image](https://user-images.githubusercontent.com/57666493/207484688-b1f0e9f2-0d49-4299-8744-fb43c785045b.png)

**Comment by tianzhou2011** [comment]
It’s more like a technical problem than a algorithm one.  I am not sure how you implement you user defined dataloader. Carefully check the saving code, pdb to set the stop point, check your saving dictionary before saving to see why additional key and value is there might help you.

**Comment by jwee-star** [comment]
well, thanks for your reply! I will check that carefully later.
